src/views/components/item.jsx

```diff
@@ -1,16 +1,56 @@
 import React from 'react';
-
+import ItemStore from '../../store/item-store.js';
+import { Link } from 'react-router';
 export default class Item extends React.Component {
+    constructor(props) {
+        super(props);
+        if (props.item !== null)
+            this.state = {item: props.item};
+        else
+            this.state = {item: null};
```

Comment:
```javascript
        if (props.item !== null)
            this.state = {item: props.item};
        else
            this.state = {item: null};
```

This is equivalent to:

```javascript
this.state = {item: props.item};
```

And the simplified version doesn't even have an ugly lack of curly fries! (@tondef001)